### DISCUSSION TITLE
Github Education 502 Bad Gateway nginx error

### DISCUSSION THREAD

**Discussion by itzme1on** [community/community#32174] [GitHub Education]
Hey, GitHub community! I am GitHub Campus member and when I try to open [education.github.com](education.github.com) I get an error 502 Bad Gateway nginx/1.15.12

![image](https://user-images.githubusercontent.com/47828778/188754246-66ad99ec-14ee-42fc-b9a6-92cfac63d35e.png)

How can I solve this problem?

**Comment by BoZenKhaa** [comment]
I am having the same problem after trying to sign up for GitHub education. I refreshed the site after waiting for a response for a long time. Now, when I log in to GitHub, I get the 502 Bad Gateway when trying to access education.github.com.

**Reply by samus-aran** [reply]
Hi @BoZenKhaa ! 👋 

Are you able to access GitHub Education again? Let us know how you get on. If you're still having issues take a screenshot of where the website is failing and lets see if we can assist 📸

**Reply by BoZenKhaa** [reply]
I just checked and after the error appeared briefly, everything seems to work nicely now! Thank you!

**Comment by mweidner037** [comment]
I'm seeing the same issue: an email linked me to https://education.github.com/ , which redirects to https://education.github.com/auth/githubber , which gives 502 Bad Gateway.
I worked around it by opening https://education.github.com/ in a private window and logging into github there.
